src/toil/test/src/jobCacheTest.py

```diff
@@ -37,6 +38,7 @@
 testingIsAutomatic = True
 
 
+# noinspection PyClassHasNoInit
```

Comment:
Can you remove this and disable the inspection in your IDE instead? I don't think this is a valid inspection in the first place. I can't see why a class not having a constructor could be a problem in Python.